core/src/main/kotlin/org/evomaster/core/problem/rest/service/RestSampler.kt

```diff
@@ -430,12 +433,18 @@ class RestSampler : AbstractRestSampler(){
 
         adHocInitialIndividuals.clear()
 
-        //init first sampling with 1-action call per endpoint, for all auths
+        if (config.seedTestCases) {
+            val parser = getParser()
+            val seededTestCases = parser.parseTestCases(config.seedTestCasesPath)
+            adHocInitialIndividuals.addAll(seededTestCases.map { createIndividual(it) })
+        } else {
```

Comment:
I do not think this should be an `else`.
the regular initialization should still be done. it is just that, if `config.seedTestCases` is true, then we add even more test cases

Reply:
Done.